README.md

```diff
@@ -1,5 +1,5 @@
 # Open Source Databases Tools Service 
-The PostgreSQL Tools Service is an application that provides core functionality for various PostgreSQL Server tools.  These features include the following:
+The Open Source Databases Tools Service is an application that provides core functionality for various PostgreSQL and MySQL Server tools.  These features include the following:
```

Comment:
After we change the name to ossdbtoolsservice, we have to change the links below:







[GitHub Issue Tracker]:https://github.com/Microsoft/pgtoolsservice/issues
[bug report]:https://github.com/Microsoft/pgtoolsservice/issues/new?labels=bug